androidApp/src/main/java/com/mbta/tid/mbta_app/android/MainApplication.kt

```diff
@@ -34,6 +36,7 @@ class MainApplication : Application() {
 
     companion object {
         val koinViewModelModule = module {
+            single<Analytics> { AnalyticsProvider.shared }
             viewModelOf(::ContentViewModel)
             viewModelOf(::NearbyTransitViewModel)
```

Comment:
I'm injecting the `Analytics` via Koin so that I don't have to pass it as an explicit argument to the `NearbyTransitViewModel` constructor; this may be more trouble than it's worth.

Reply:
Injecting the `Analytics` via Koin turns out to actually be a really good idea because it's a lot harder to mock out the analytics for testing without doing that.